Xamarin.Forms.Platform.iOS/Renderers/PageRenderer.cs

```diff
@@ -80,7 +80,7 @@ public override void ViewDidAppear(bool animated)
 			UpdateStatusBarPrefersHidden();
 		}
 
-		public override void ViewDidDisappear(bool animated)
+		public override async void ViewDidDisappear(bool animated)
```

Comment:
So what's a workaround for this? I need to await `PopAsyncInner` or else we end up with the same issue. Changing the return signature of `ViewDidDisappear` is also not an option.